.borp.int.yaml

```diff
@@ -3,3 +3,4 @@ files:
   # We can't do a simple '**/*.test.js' because the "uninstrumented" suite
   # includes a `node_modules` directory that includes several `.tests.js` files.
   - 'test/integration/*.test.js'
+  - 'test/integration/config/*.test.js'
```

Comment:
No. The comment directly preceding these entries explains why.

Reply:
oh, they are in node_modules. I'd prefer we add a glob to exclude those instead of specifying every folder we have in integration tests, this will become unwieldy

Reply:
@bizob2828 okay, we'll need https://github.com/mcollina/borp/pull/127 to be released in order to do that.

Reply:
i won't hold this up. we can also upgrade and remove these entries